packages/widget/src/hooks/useAutoSetAddress.ts

```diff
@@ -60,11 +61,11 @@ export const useAutoSetAddress = () => {
         if (!chain) {
           return;
         }
+        if (chainAddresses[index]?.address) return;
```

Comment:
Made a small code improvement here. Its better to just handle this at top of hook instead of 3 times for each case? 
@codingki